web/src/pages/summary.rs

```diff
@@ -22,7 +22,21 @@ impl Component for Summary {
         html! { <>
             <h1> { "Summary" } </h1>
 
-            <p> { "We showed that AT2 is faster than bitcoin." } </p>
+            <p> { "We showed that AT2 is faster than other well known
```

Comment:
As discussed, please start a google-doc and invite me and Christian to edit it. This should be much longer and explain more in-depth what is happening. It should also link directly to the paper and perhaps other resources.

Reply:
Shared in engineers' chat